src/test/java/uk/gov/hmcts/divorce/citizen/notification/Applicant2RequestChangesNotificationTest.java

```diff
@@ -192,6 +195,34 @@ void shouldSendEmailToApplicant2WithDivorceContent() {
         verify(commonContent).mainTemplateVars(data, 1234567890123456L, data.getApplicant2(), data.getApplicant1());
     }
 
+    @Test
+    void shouldSendEmailToApplicant2WithDivorceContentWhenLangPrefIsWelsh() {
```

Comment:
Is this test needed now that the decision on setting Welsh content is inside CommonContent, and that is mocked here?